source/Generic/PlayState/PlayState.cs

```diff
@@ -494,6 +488,12 @@ private string GetHoursString(ulong playtimeSeconds)
         /// </summary>
         private void ShowNotification(string status)
         {
+            showWindowsNotificationsStyle = false;
```

Comment:
can be simplified as

```csharp
showWindowsNotificationsStyle = settings.Settings.GlobalShowWindowsNotificationsStyle
```

Also, is it needed to store the value in the plugin itself? Or can it be a variable inside this method like

```csharp
var showWindowsNotificationsStyle = settings.Settings.GlobalShowWindowsNotificationsStyle
```

Reply:
I did that way for following the previous code, but the second case you said I think it's better. I will use it.